Odata-docs/webapi-8/getting-started.md

```diff
@@ -0,0 +1,381 @@
+---
+title:  "Getting Started with ASP.NET Core OData 8"
+description: A getting started guide to creating and running an ASP.NET Core OData 8 application.
+date:   2022-11-11
+ms.date: 11/11/2022
+author: gathogojr
+ms.author: jogathog
+---
+
+# Getting Started with ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v8.md)]
+
+This tutorial shows how to create and run an ASP.NET Core OData 8 application
+
+You'll learn how to:  
+:white_check_mark: Create an ASP.NET Core application  
+:white_check_mark: Add Microsoft.AspNetCore.OData library package  
+:white_check_mark: Define the CLR types  
+:white_check_mark: Add an OData controller  
+:white_check_mark: Build the Edm model and configure the service  
+:white_check_mark: Run the OData service  
+:white_check_mark: Interact with the OData service from an API client  
+
+## Prerequisites
+- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/#download) with the ASP.NET and web development workload - if target framework is [.NET Core 3.1](https://dotnet.microsoft.com/en-us/download/dotnet/3.1) or [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), or
+- [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/) with the ASP.NET and web development workload - if target framework is [.NET Core 3.1](https://dotnet.microsoft.com/en-us/download/dotnet/3.1) 
```

Comment:
Given that .NET Core 3.1 is [going out of support on December 13th](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle), I don't think we should encourage it for new applications. If we want to document support for .NET Core 3.1, I think we should provide examples in the Guides section or another section outside the "Getting started" section.